src/test/scala/scalikejdbc/athena/ConfigSpec.scala

```diff
@@ -7,22 +7,25 @@ class ConfigSpec extends FunSpec with OptionValues {
   describe("Config") {
     it("default db") {
       val config = new Config("default")
-      assert(config.url === "jdbc:awsathena://athena.us-east-2.amazonaws.com:443")
-      assert(config.options.getProperty("s3_staging_dir") === "s3://query-results-bucket/folder/")
-      assert(config.options.getProperty("log_path") === "logs/application.log")
+      assert(config.url === "jdbc:awsathena://AwsRegion=us-east-2")
+      assert(config.options.getProperty("S3OutputLocation") === "s3://query-results-bucket/folder/")
+      assert(config.options.getProperty("LogPath") === "logs/application.log")
       assert(config.getTmpStagingDir.isEmpty)
+      assert(config.readOnly === Some(false))
```

Comment:
In the same way as `getTmpStagingDir` assertion,
please use `config.readOnly.value`, `config.readOnly.isEmpty`.